src/aria/widgets/controllers/TextDataController.js

```diff
@@ -222,6 +256,104 @@ module.exports = Aria.classDefinition({
                 report : report,
                 arg : arg
             });
+        },
+
+        /**
+         * Retrieves the requested error message from different sets of configuration ordered by precedence.
+         *
+         * <p>
+         * An error message value is requested from an this error message name.
```

Comment:
"this" actually, since it refers to the error massage mentioned at the beginning of the sentence.